README.md

```diff
@@ -4,31 +4,37 @@
 
 ### Prerequisites
 
-Make sure the Qt (>= 5.3.2) development libraries are installed:
+Make sure the Qt (>= 5.3.2 and < 5.15) development libraries are installed:
```

Comment:
5.4.2 <= ... <= 5.15?
I use 5.15 and 5.4.2 locally on Linux, so I know that it works for these OSes.

How do you know that UGENE builds with 5.3.2?

Reply:
I've got an qt assert when I have tried to run UGENE with 5.15 on Windows. I will fix max version.
I do not know about 5.3.2, it just was here. I can change it to 5.4.2.